<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana7</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>901</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ImagenCruces.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>479</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo_8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>279</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>279</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganador_Final</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #28a745;  /* Color de fondo verde */
    color: white;               /* Color del texto */
    border-radius: 5px;         /* Bordes redondeados */
    padding: 5px;               /* Espacio interno */
}
QPushButton:hover {
    background-color: #218838;  /* Color cuando el mouse pasa encima */
}
QPushButton:pressed {
    background-color: #1e7e34;  /* Color cuando el bot√≥n es presionado */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgbot8.png</normaloff>imgbot8.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>90</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>siguiente_pag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>siguiente_pag()</slot>
 </slots>
</ui>
